### DISCUSSION TITLE
[PROPOSAL] Gobbit SFT

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2416] [General]
Art from Bunny Trove:
https://github.com/sunflower-land/sunflower-land/raw/main/src/assets/events/easter/2023/npcs/gobbit.gif

Boost:
When harvesting a carrot crop, harvests up to 5 adjacent carrot plots at once.

Notes:
- I'd be fine with the same number of chest prompts (if needed) as today; though coalescing those (up to) 5 into a single would be nice too.
- A sunflower version would be heavenly.
- Consider doing something similar with the beefcake from the old Goblin place.
- Could have a small army of these -- 1 for each crop -- as part of season 3 or something.  Let people decide which ones they want to go for.

**Comment by DaanMich** [comment]
Would like this a lot! 
Personally I'd be fine if this was a high supply SFT with a high cost as well. 

I feel like crops don't get as much love for mints so would love a goal to strive for and mint this gigantic bunny
